corona-warn-app/cwa-app-android
# Adjust vaccination list screen (EXPOSUREAPP-7238)

Main adjustments of the vaccination list screen are done. 

### How to test: 
- scan Vaccinations from ImpfzertifikatBeispiele.zip
- check different states Incomplete / Complete / Immunity - maybe you have to change the phone date to see the complete / Immunity state

### Incomplete: 
![image](https://user-images.githubusercontent.com/10398034/118627469-09c44080-b7cc-11eb-8fd1-004db49ff946.png)

### Complete: 
![image](https://user-images.githubusercontent.com/10398034/118627717-3ed09300-b7cc-11eb-8450-dd73ec42e044.png)


### Immunity: 
![image](https://user-images.githubusercontent.com/10398034/118627555-1f396a80-b7cc-11eb-98ea-6ae526bab137.png)
